superset-frontend/src/views/components/SubMenu.tsx

```diff
@@ -111,8 +112,9 @@ const StyledHeader = styled.div`
           ${({ theme }) => theme.gridUnit * 4}px;
       }
 
-      &.active a {
-        text-decoration: underline;
+      &.active {
```

Comment:
```suggestion
      &.active a {
```
I think you want this `a` back here, so these styles are applied to the link rather than the div encapsulating it. This makes them the same apparent height.